webapp-django/crashstats/crashstats/middleware.py

```diff
@@ -3,16 +3,17 @@
 
 class SetRemoteAddrFromForwardedFor(object):
```

Comment:
Shouldn't we rename the middleware, since it no longer looks at ForwardedFor?

Reply:
Yes! I'll do that now.